lib/parser.js

```diff
@@ -0,0 +1,21 @@
+var falafel = require('falafel');
+var esprima = require('esprima');
+
+module.exports = {
+  walk: function(js, callback) {
+    return falafel(js, {
+      parser: this
+    }, function(node) {
+      node.start = node.range[0];
```

Comment:
FYI: This is to make the esprima tree compatible with falafel's update fn's. Not sure if this sort of thing should be upstreamed to falafel.